uniffi_bindgen/src/bindings/swift/gen_swift.rs

```diff
@@ -18,36 +16,64 @@ use crate::MergeWith;
 
 #[derive(Debug, Clone, Default, Serialize, Deserialize)]
 pub struct Config {
-    module_name: Option<String>,
     cdylib_name: Option<String>,
+    module_name: Option<String>,
+    ffi_module_name: Option<String>,
+    header_filename: Option<String>,
+    generate_module_map: Option<bool>,
 }
+
 impl Config {
+    /// The name of the Swift module containing the high-level foreign-language bindings.
     pub fn module_name(&self) -> String {
         match self.module_name.as_ref() {
             Some(name) => name.clone(),
             None => "uniffi".into(),
         }
     }
+
+    /// The name of the lower-level C module containing the FFI declarations.
+    pub fn ffi_module_name(&self) -> String {
+        match self.ffi_module_name.as_ref() {
+            Some(name) => name.clone(),
+            None => format!("{}FFI", self.module_name()),
+        }
+    }
+
+    /// The name of the `.modulemap` file for the lower-level C module with FFI declarations.
     pub fn modulemap_filename(&self) -> String {
-        format!("{}.modulemap", self.module_name())
+        format!("{}.modulemap", self.ffi_module_name())
     }
+
+    /// The name of the `.h` file for the lower-level C module with FFI declarations.
     pub fn header_filename(&self) -> String {
-        format!("{}-Bridging-Header.h", self.module_name())
```

Comment:
I've disliked this `-Bridging-Header.h` suffix for a long time. IIUC it's like this because that's the default that Xcode generates when you ask it to make a bridging header for you, but IMHO it's more confusing than helpful in our case, leading the consumer to ask questions like "what is a bridging header?" and "do I need to do something special with bridging headers?" when really they just need to treat it like an ordinary header.